<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QCheckBox" name="newNameCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>new name</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>suffix:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>prefix:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="renameCheckBox">
     <property name="text">
      <string>rename</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>counter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>counter start:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="counterCheckBox">
        <property name="text">
         <string>use counter</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>counter decimals:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="startSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="decimalSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="suffixLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="prefixLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="newNameLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QLabel" name="exampleLabel">
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>example</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>newNameCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>newNameLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>counterCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>startSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>counterCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>decimalSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>renameCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>renameCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>prefixLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>renameCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>suffixLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>renameCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>newNameCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>renameCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>newNameLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
